bin/install-all.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu
+
+kubectl create namespace securecodebox-system
+helm -n securecodebox-system install securecodebox-operator ./operator/
```

Comment:
I would recommend installing the operator with "helm -n securecodebox-system upgrade --install securecodebox-operator ./operator/" as well

Reply:
Done